values-prometheus.yaml

```diff
@@ -12,3 +11,45 @@ nodeExporter:
 
 pushgateway:
   enabled: false
+
+# extra scraping configs
+# | is required, because extraScrapeConfigs is expected to be a string
+extraScrapeConfigs: |
+  - job_name: 'kubernetes-service-endpoints-scrape-every-2s'
+
+    scrape_interval: 2s
+
+    kubernetes_sd_configs:
+      - role: endpoints
```

Comment:
My guess is that you should base this on the [pods scrape config](https://github.com/helm/charts/blob/91d0b7e3c8d319313db621622f29332edf8ab633/stable/prometheus/values.yaml#L1474-L1511) instead of endpoints, unless you had a particular reason to base it on the endpoints config.

Reply:
switched to pods